README.md

```diff
@@ -58,14 +57,24 @@ node_templates:
     image:
         type: im.nodes.Image
         properties:
-            name: Centos-/
+            name: Centos-7
             config: 
                 id: 'https://fedcloud-services.egi.cesga.es:11443/51'
                 storage: 1024M
-                username: ...
-                password: ...
-                public_key: 'ssh-rsa ...'
-                private_key: |
+                username: ... # Optional
```

Comment:
How is the username optional? I guess we must always know the default user of the image

Reply:
The `username` is optional from the blueprint point of view. From the plugin point of view, if a `username` is not provided, `user` is used by default

Reply:
But it doesn't make sense, it is very unlikely that, if you left the username empty, the user in the image will be 'user'.

Therefore I think that the username should be mandatory.